spec/records/edit_fields/_rights.html.erb_spec.rb

```diff
@@ -25,15 +25,10 @@
     render partial: "records/edit_fields/rights", locals: { f: f }
   end
 
-  it "rights should have asides" do
-    num_asides = RightsService.select_options.length
-    expect(rendered).to have_selector(".generic_work_rights .radio aside", count: num_asides)
-  end
-
   it "rights should have descriptions for each right" do
     num_asides = RightsService.select_options.length
     RightsService.select_options.each do |right|
-      expect(rendered).to have_content(t_uri(:description, scope: [ :rights, right[1] ]))
+      expect(rendered).to have_content(t_uri(:description, scope: [ :rights, right[1] ]))if !right[1].include?('3.0')
```

Comment:
This is one of the reasons to make the change in the RightsService.  When we add rights 4.1, 4.2, ect.. we'd have to update the logic in the ui that uses it and add the logic into the tests.  This conditional will eventually end up as a compound one and finally a regex before it becomes too difficult to update.